packages/toolpad-core/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "tsup",
+    "build": "tsup && tsc --emitDeclarationOnly --declaration",
```

Comment:
Not sure why it worked on Mac but `tsup` doesn't generate type declaration files. https://tsup.egoist.dev/#generate-typescript-declaration-maps--d-ts-map

Reply:
We run the type generation in the `onsuccess` handler (https://github.com/mui/mui-toolpad/blob/bebe8a36c43dc13388d19ddad841aab1975c3893/packages/toolpad-core/tsup.config.ts#L14)

Are you sure the build is finishing successfully?

Reply:
I hadn't checked yet but yeah, if it works on Mac I guess the types definitions were being generated there. This explains it, should have searched for `emitDeclarationOnly` :)
So I guess the issue must be something like this `spawnSync` not running on Windows. I'll take a look at that when I'm back.

Reply:
I think you can remove these `onSuccess` handlers and just turn on `dts: true`. Looks like it was implemented this way to improve performance as per https://github.com/egoist/tsup/issues/615. But we don't have `incremental` enabled (anymore?) and I don't notice a big problem in performance.